api/PointerEvent.json

```
@@ -492,28 +492,48 @@
             "description": "Fractional coordinates for <code>mouse</code>.",
             "support": {
               "chrome": {
-                "version_added": "64"
+                "version_added": "64",
+                "partial_implementation": true,
+                "notes": "<code>movementX</code> and <code>movementY</code> are not fractional, see <a href='https://crbug.com/802067'>Chromium bug 802067</a>."
               },
               "chrome_android": {
-                "version_added": "64"
-              },
-              "edge": {
-                "version_added": "≤79"
+                "version_added": "64",
+                "partial_implementation": true,
+                "notes": "<code>movementX</code> and <code>movementY</code> are not fractional, see <a href='https://crbug.com/802067'>Chromium bug 802067</a>."
               },
+              "edge": [
+                {
+                  "version_added": "79",
+                  "partial_implementation": true,
+                  "notes": "<code>movementX</code> and <code>movementY</code> are not fractional, see <a href='https://crbug.com/802067'>Chromium bug 802067</a>."
+                },
+                {
+                  "version_added": "12",
+                  "version_removed": "79",
+                  "partial_implementation": true,
+                  "notes": "Only <code>clientX</code>, <code>clientY</code>, <code>pageX</code> and <code>pageY</code> are fractional."
+                }
+              ],
               "firefox": {
-                "version_added": null
+                "version_added": false
               },
               "firefox_android": {
                 "version_added": null
               },
               "ie": {
-                "version_added": null
+                "version_added": true,
+                "partial_implementation": true,
+                "notes": "Only <code>clientX</code>, <code>clientY</code>, <code>pageX</code> and <code>pageY</code> are fractional."
               },
               "opera": {
-                "version_added": "51"
+                "version_added": "51",
+                "partial_implementation": true,
+                "notes": "<code>movementX</code> and <code>movementY</code> are not fractional, see <a href='https://crbug.com/802067'>Chromium bug 802067</a>."
               },
               "opera_android": {
-                "version_added": "47"
+                "version_added": "47",
+                "partial_implementation": true,
+                "notes": "<code>movementX</code> and <code>movementY</code> are not fractional, see <a href='https://crbug.com/802067'>Chromium bug 802067</a>."
               },
               "safari": {
                 "version_added": null
```

I tested this in Safari, and it looks like fractional values aren't supported at all.

```suggestion
                "version_added": false
```

---

Thanks! Just to confirm, you tested this on a local device, because at least through browserstack I always got integers too (at least for Chrome).

---

I did, yeah -- my main computer's a MacBook Pro with the latest macOS.  :wink: